config/default/translations/messages-en.properties

```
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+
+report.pregnancy_home_visit.pregnancy_ended = Update Pregnancy
+report.pregnancy_home_visit.pregnancy_ended.miscarriage_note =
+report.pregnancy_home_visit.pregnancy_ended.miscarriage_date = Date of miscarriage
+report.pregnancy_home_visit.pregnancy_ended.abortion_date = Date of abortion
+report.pregnancy_home_visit.pregnancy_ended.clear_option = Clear option
+
+report.pregnancy_home_visit.update_g_age = Update Pregnancy
+report.pregnancy_home_visit.update_g_age.update_method = Gestational Age Update Method
+report.pregnancy_home_visit.update_g_age.update_method.g_age_update_method =  Gestational age update method
+report.pregnancy_home_visit.update_g_age.update_method.lmp_weeks_new = Current weeks pregnant.
+report.pregnancy_home_visit.update_g_age.update_method.u_edd_new = New EDD.
+report.pregnancy_home_visit.update_g_age.update_method.update_method = Update Method
+report.pregnancy_home_visit.update_g_age.update_summary = Update Summarry
+report.pregnancy_home_visit.update_g_age.previous_g_age = Weeks pregnant
+report.pregnancy_home_visit.update_g_age.previous_g_age_unknown = Unknown weeks pregnant
+report.pregnancy_home_visit.update_g_age.previous_edd = EDD: 
+report.pregnancy_home_visit.update_g_age.previous_edd_unknown = EDD: unknown
+report.pregnancy_home_visit.update_g_age.new_info = New
+report.pregnancy_home_visit.update_g_age.new_g_age = Weeks pregnant
+report.pregnancy_home_visit.update_g_age.new_edd = EDD: 
+report.pregnancy_home_visit.update_g_age.lmp_date_8601_new = LMPD DATE 8601 NEW 
+report.pregnancy_home_visit.update_g_age.lmp_date_new = LMP DATE
+report.pregnancy_home_visit.update_g_age.edd_8601_new = NEW EDD 8601
+report.pregnancy_home_visit.update_g_age.edd_new = NEW EDD
+
+report.pregnancy_home_visit.anc_visits_hf = Health facility ANC visits
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past = ANC Visits at Health Facility (Past)
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.pregnancy_follow_up_date_recent = Recent Follow up date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.last_visit_attended = Woman completed the health facility ANC visit as scheduled
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.report_other_visits = Report any additional unreported health facility ANC visits
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_hf_count =  Number of additional unreported health facility ANC visits
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_date_ask_single = Facility Visit date 
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_date_single = Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates = Vist Dates
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates_count = Number of visit Dates
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.0 = First Visit Date
```

Should this be sentence case?

```suggestion
report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.0 = First visit date
```